week-1/errors/1.js

```diff
@@ -1,4 +1,5 @@
 // trying to create an age variable and then reassign the value by 1
 
-const age = 33;
-age = age + 1;
+let age = 33; //variable created with "const" is constant and unchangeable. We fix this by reassigning the variable with "let"
+age++; // We can increase the variable by using age = age + 1 or simply using age++ as done here
+console.log(age);
```

Comment:
Always try to remember to clean up any extra console.log statements if they aren't required by the instructions